### PR TITLE
MINOR: update javadoc on ReadOnlyWindowStore

### PR DESCRIPTION
**Pull request by dguy**
Highlight that the range in `fetch` is inclusive of both `timeFrom` and `timeTo`

### CODE REVIEW

**Comment by dguy** [comment]
@mjsax @enothereska @miguno

**Review by miguno — APPROVED** [review]

**Review by mjsax — APPROVED** [review]
Thanks Damian!

**Comment by mjsax** [comment]
@ijuma Can you cherry-pick this to `0.10.2`, too?

File: streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java [hunk 1]
```diff
@@ -32,7 +32,26 @@
...
+     * +-------+------------+----------+
+     * |   A   |     15     |    25    |
+     * +-------+------------+----------+
+     * |   A   |     20     |    30    |
+     * +-------+------------+----------+
+     * |   A   |     25     |    35    |
+     * +--------------------------------
+     * </pre>
+     * And we called {@code store.fetch("A", 10, 20)} then the results will contain the first
+     * three windows from the table above, i.e., all those were start time >= 10 and <= 20.
```
Comment (ijuma) [inline streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java]: Minor typo, `were` -> `where`? Also, `10 <= start time <= 20` might be slightly better notation.

File: streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java [hunk 2]
```diff
@@ -32,7 +32,26 @@
...
+     * |   A   |     10     |    20    |
+     * +-------+------------+----------+
+     * |   A   |     15     |    25    |
+     * +-------+------------+----------+
+     * |   A   |     20     |    30    |
+     * +-------+------------+----------+
+     * |   A   |     25     |    35    |
+     * +--------------------------------
+     * </pre>
+     * And we called {@code store.fetch("A", 10, 20)} then the results will contain the first
```
Comment (ijuma) [inline streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java]: `called` -> `call`?

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM with some minor changes. If you're OK with them, I'll make them during the merge.

**Comment by ijuma** [comment]
Merged to trunk and 0.10.2.

**Pull request closed without merging**